css/style.css

```diff
@@ -17,15 +17,15 @@ body {
 .header_color_black {
 	display: flex;
 	width: 31.5vw;
-	height: 92px;
+	height: 100px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_A/pullRequest?prid=6206892)